core/src/main/java/ai/timefold/solver/core/api/domain/common/SorterWeightFactory.java

```diff
@@ -0,0 +1,31 @@
+package ai.timefold.solver.core.api.domain.common;
+
+import ai.timefold.solver.core.api.domain.entity.PlanningEntity;
+import ai.timefold.solver.core.api.domain.solution.PlanningSolution;
+import ai.timefold.solver.core.impl.heuristic.move.Move;
+import ai.timefold.solver.core.impl.heuristic.selector.Selector;
+
+/**
+ * Creates a weight to decide the order of a collections of selections
+ * (a selection is a {@link PlanningEntity}, a planningValue, a {@link Move} or a {@link Selector}).
```

Comment:
Where do we weight selectors?
(And if selectors are not public API, should this be mentioning them?)